ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -0,0 +1,54 @@
+base_images:
+  stackrox-test:
+    name: stackrox-test
+    namespace: stackrox
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: stackrox-test
+    namespace: stackrox
+    tag: latest
+releases:
+  latest:
+    integration:
+      name: "4.8"
```

Comment:
Isn't OpenShift 4.10 the latest GA release?

Reply:
Don't know and tbh I don't know how this stanza affects CI but it is required, so I copied what is used in other places.